ansible/roles/init/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
 
 - name: Add ppa Repository
   become: yes
-  apt_repository: repo=ppa:ondrej/{{ php_ppa }}
```

Comment:
The problem is if we change it here the source of the variable will be useless and bugs may arise from that.